pubsublite/internal/wire/committer.go

```diff
@@ -228,9 +251,16 @@ func (c *committer) unsafeInitiateShutdown(targetStatus serviceStatus, err error
 }
 
 func (c *committer) unsafeCheckDone() {
```

Comment:
Nit; do you mind adding a short doc explaining what this function does? It took me a while to figure it out.

Reply:
Done